src/keys/public.rs

```diff
@@ -54,3 +64,46 @@ impl Serializable<32> for PublicKey {
         Ok(Self(JubJubAffine::from_bytes(bytes)?.into()))
     }
 }
+
+mod tests {
+
+    #[test]
+    fn partial_eq_test() {
+        use super::*;
+        use dusk_jubjub::{JubJubAffine, JubJubScalar};
+        use rand_core::OsRng;
+
+        let sk1 = PublicKey::from(&SecretKey::random(&mut OsRng));
+        let sk2 = PublicKey::from(&SecretKey::random(&mut OsRng));
+
+        assert_eq!(sk1, sk1);
```

Comment:
What's the point to do this?

Reply:
This is a simple assertion that the same key equals itself. Since the implementation of the equality is non-trivial (e.g. its a formula evaluation), its good to cover also the basic case.

Reply:
Ok. Please add `#[allow(clippy::eq_op)]` to the test then, and move it as integration test and not as unit test (you're not testing any internal here, you're testing public types' behavior).